src/main/kotlin/com/atlassian/performance/tools/awsinfrastructure/api/InfrastructureFormula.kt

```diff
@@ -59,16 +72,17 @@ class InfrastructureFormula<out T : VirtualUsers>(
         }
 
         val network = networkProvisioning.get()
-        val provisionJira = executor.submitWithLogContext("jira") {
-            overrideJiraNetwork(network).provision(
+
+        val provisionedJira =  userProvisionedJira ?: executor.submitWithLogContext("jira") {
+            overrideJiraNetwork(network)!!.provision(
                 investment = investment,
                 pluginsTransport = aws.jiraStorage(nonce),
                 resultsTransport = resultsStorage,
                 key = keyProvisioning,
                 roleProfile = roleProfile,
                 aws = aws
             )
-        }
+        }.get()
```

Comment:
This change breaks the concurrency of Jira and VU provisioning. We don't have a test for `InfrastructureFormula` so CI doesn't catch this at the moment. We gotta revert this

Reply:
PS. such test is included in a [WIP side-branch](https://github.com/atlassian/aws-infrastructure/compare/master...dagguh:issue/JPERF-554-kibana-results#diff-19cab7e2eed0cf07a440956071c59558)

Reply:
I'm going to fix that.